scripts/rebuildstaging.py

```diff
@@ -23,31 +23,30 @@
 
 When not specified, a submodule's trunk and name inherit from the parent
 """
-from __future__ import print_function
-from __future__ import absolute_import
+from __future__ import absolute_import, print_function, unicode_literals
 
-from __future__ import unicode_literals
-from gevent import monkey
-import six
-monkey.patch_all()
-
-import contextlib
 import os
 import re
-import sh
 import sys
+from contextlib import ExitStack
 
 import gevent
+import jsonobject
+import sh
+import six
+from gevent import monkey
+
 from gitutils import (
     OriginalBranch,
     get_git,
     git_recent_tags,
     has_merge_conflict,
     print_merge_details,
 )
-import jsonobject
 from sh_verbose import ShVerbose
 
+monkey.patch_all()
```

Comment:
Is moving ```monkey.patch_all()``` intentional? Usually it should be done as soon as possible in a file, even if it goes against import formatting best practices.

Reply:
updated, thanks for flagging